### PR TITLE
Preparation for chaos...

### PR DESCRIPTION
**Pull request by msrb**
Groundwork for https://github.com/openshiftio/openshift.io/issues/1084.

### CODE REVIEW

File: integration-tests/Dockerfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ ENV LANG=en_US.UTF-8 \
     NOVENV=1
 
 RUN yum install -y epel-release &&\
-    yum install -y gcc which python34-pip python34-devel &&\
+    yum install -y gcc which python34-pip python34-devel centos-release-openshift-origin &&\
```
Comment (jpopelka) [inline integration-tests/Dockerfile]: Just FYI: We had some (can't remember exactly) problems with the latest `centos-release-openshift-origin`, so we [use centos-release-openshift-origin36 in deployment](https://github.com/fabric8-analytics/fabric8-analytics-deployment/blob/master/openshift/Dockerfile#L8)

File: integration-tests/features/src/utils.py [hunk 2]
```diff
@@ -14,3 +15,52 @@ def download_file_from_url(url):
...
+    """
+    command = ['oc', 'login', url, '--username', username, '--password', password]
+    if not tls_verify:
+        command.extend(['--insecure-skip-tls-verify=true'])
+
+    try:
+        subprocess.check_call(command)
+    except subprocess.CalledProcessError as e:
+        # replace password with '***' so somebody will not accidentally leak it in CI logs
+        e.cmd = [x if x != password else '***' for x in e.cmd]
```
Comment (jpopelka) [inline integration-tests/features/src/utils.py]: It won't work if the password is part of longer string, therefore:

```python
e.cmd = [x.replace(password, '***') for x in e.cmd]
```

File: integration-tests/Dockerfile (hunk 1 above)
Comment (msrb) [reply integration-tests/Dockerfile]: Updated :+1:

File: integration-tests/features/src/utils.py (hunk 2 above)
Comment (msrb) [reply integration-tests/features/src/utils.py]: Your version is simpler and easier to read, updated. Thanks :+1:

**Comment by tisnik** [comment]
It looks good @msrb, thank you. Let the chaos begin!

**Pull request merged**